### DISCUSSION TITLE
Calculating skew quadrupole strengths

### DISCUSSION THREAD

**Discussion by elafmusa** [atcollab/at#769] [Python AT]
Hi 

I have a basic question, when introducing shifts to sextuple elements, how can I compute the resulting skew quadrupole strengths that arise from these errors? E.g. to be used in RDTs formulas.

Thank you in advance

**Comment by lcarver** [comment]
The formula for a sextupole kick is:

```
x' = b3 ( x^2 - y^2)
y' = -2 b3 x y
```

if you assume a vertical offset (to give you skew quad)

`y' = -2 b3 x (y + dy) = -2 b3 x y - 2 b3 x dy`

So the skew quadrupole component is `-2 b3 dy`
